project_report/tex/evaluation.tex

```diff
@@ -1 +1,53 @@
+The group has managed to meet the requirements, set in the initial requirements
+analysis, in the beginning of the project. The created system is able to control
+the production line, using the dashboard, and is able to collect, display and
+store relevant data in a database. The data is used for further analysis, such
+as calculating the OEE and optimal machine speed, as well as estimating the
+error function.
+The design of the dashboard, makes it easy to see what is going on the front-page.
+Every item on the front-page is there nothing is hiding down under a sub menu,
+beside starting the beer machine, you would need to press the start button,
+that will then take you to a start-page where you can chose a recipe, speed and 
+the amount to produce. The start-page is not that convenient, because the page 
+does not put in the recommend values for the different recipes, but it does
+show the recommend values for the recipe.
+For seeing other older batches you will need to navigate to a different page
+called history.html. You can navigate to this page easy from the front-page by
+pressing on the button left side menu bar that takes you to the history page.
+Where you will be able to chose a batch to inspect.
+The button layout is the same as the front-page, so there should be no learning
+curve there to check out the old batches.
+Overall the group thinks it is a good system for people not familiar with the
+project, because it does comply with the standard UI formalities and there is
+almost no learning curve to the dashboard.
+
+
+\subsection{Development process}
+In the beginning of the development process, the group has held as meeting
+about how the projects should be structured. Building on the experience from
+last semester where Unified Process and Scrum was used, the group
+decided to use both UP and Scrum again. Every Friday, a Scrum meeting is held,
+to show and discuss what each group member has accomplished. Each task is
+assigned into a two week sprint, which is planned at the appropriate Scrum
+meeting.
+During 'matching of expectations', the group agreed upon focusing on the Minimum
+Viable Product, meaning not implementing parts outside of the semester project's
+scope. This has been a more realistic goal in comparison to last semester.
+The use of Scrum has giving a great overview of the project, and has made it 
+easier to plan ahead. However, looking back, the group should have scheduled 
+more in beginning. Near the end, the group has had longer work hours.
+
+The project has been divided into the four phases Unified Process; Inception,
+Elaboration, Construction, and Transition. Each phase has been a fixed boundary
+to plan around, making it easier to schedule, sprints, tasks, and issues.
+Unlike the previous semester, no inception document was created. Instead, the
+initial requirements analysis was incorporated directly into the final report.
+The initial phases of Unified Process could have been more efficient, enabling
```

Comment:
Consider rewriting, as the initial phase of UP cannot be more efficient, but how we worked with it could.